src/styles/blocks/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  font-family: "Roboto", Helvetica, sans-serif;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  &__img {
+    display: block;
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    margin: 40px 0 0;
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 150%;
+    &--code {
+      color: $secondary;
+      font-size: 10px;
+      font-style: normal;
+      font-weight: 400;
+      line-height: 140%;
+    }
+  }
+
+  &__raiting {
+    margin-top: 16px;
+    align-items: center;
+    justify-content: space-between;
+    display: flex;
+  }
+
+  &__reviews {
+    margin: 0;
+    color: $main-accent;
+    font-size: 10px;
+    font-weight: 400;
+  }
+
+  &__price {
+    margin-top: 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__word {
+    color: $secondary;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 400;
+  }
+
+  &__cost {
+    text-align: right;
+    font-family: "Roboto", Helvetica, sans-serif;
```

Comment:
As long as you have the same font family everywhere consider moving it to some top-level selector